pkg/interface/src/views/landscape/components/NewChannel.tsx

```diff
@@ -31,33 +35,38 @@ type FormSchema = {
   moduleType: 'chat' | 'publish' | 'link';
 } & ChannelWriteFieldSchema;
 
-const formSchema = (members?: string[]) => Yup.object({
-  name: Yup.string(),
-  description: Yup.string(),
-  ships: Yup.array(Yup.string()),
-  moduleType: Yup.string().required('Must choose channel type'),
-  writers: members ? shipSearchSchemaInGroup(members) : shipSearchSchema,
-  writePerms: Yup.string()
-});
+const formSchema = (members?: string[]) =>
+  Yup.object({
+    name: Yup.string(),
+    description: Yup.string(),
+    ships: Yup.array(Yup.string()),
+    moduleType: Yup.string().required('Must choose channel type'),
+    writers: members ? shipSearchSchemaInGroup(members) : shipSearchSchema,
+    writePerms: Yup.string()
+  });
 
 interface NewChannelProps {
   api: GlobalApi;
   group?: string;
   workspace: Workspace;
+  borderRadius?: number;
+  existingMembers?: Array;
```

Comment:
Array of what? Also fwiw, I'd much prefer the style of
```typescript
type NewChannelProps = {
  api: GlobalApi;
  group?: string;
  workspace: Workspace;
  existingMembers: string[];
} & PropFunc<typeof Col>;
// in component
const { api, group, workspace, existingMembers, ...rest } = props;
// in JSX
return (
  <Col
      {...rest}
   //.... rest of components
```

because it's cleaner and allows for styled-systems affordances (short prop names, responsive arrays). PropFunc is a type helper to get the props of a functional component, it's in `~/types`

Reply:
Makes sense, thank you